app/services/unified_chat_service.py

```
@@ -848,6 +846,40 @@ async def _gather_context_data(
             context_data["market_risk"] = {"factors": [], "error": "Market risk integration pending"}
             
+        elif intent == ChatIntent.STRATEGY_MANAGEMENT:
+            user_strategies = context_data.get("user_strategies", {})
+            marketplace_strategies = context_data.get("marketplace_strategies", {})
+
+            if user_strategies.get("success", False):
+                active_strategies = user_strategies.get("active_strategies", [])
+                total_strategies = user_strategies.get("total_strategies", 0)
+                total_monthly_cost = user_strategies.get("total_monthly_cost", 0)
+
+                return f"""User asked: "{message}"
```

Ruff (0.13.1)

Undefined name `message`

(F821)